libs/frontend/ui-widgets/src/lib/InfoBubble.vue

```diff
@@ -131,6 +131,7 @@ const hideBubble = () => {
   @mixin size 20px;
   cursor: pointer;
   position: relative;
+  flex-shrink: 0;
```

Comment:
Hi @sampullman 

I noticed that the inherited from info bubble is invisible when style row is too wide, so I add `flex-shrink: 0` here to make it visible again.

![Screenshot 2023-11-17 at 5 55 14 PM](https://github.com/samatechtw/pubstudio-builder/assets/59676941/2eec0952-54ea-44ac-9071-0af280267341)

Reply:
Thanks! I think there are probably a few more similar bugs here and there.